genai-perf/genai_perf/parser.py

```diff
@@ -1132,8 +1149,7 @@ def _add_tokenizer_args(parser):
     tokenizer_group.add_argument(
         "--tokenizer",
         type=str,
-        default=DEFAULT_TOKENIZER,
-        required=False,
+        required=True,
```

Comment:
I think for many users, model name and tokenizer name are the same since both are coming from the same hf repo. Instead of making it required, would it make more sense for gap to first try fetching the tokenizer from hf repo using the model name, and raise an error if it cannot find one? This will remove the need to specify both model and tokenizer for casual users who are not using local tokenizer.

Reply:
Great point. I thought about this and don't know why I ended up not adding it.

Let me do that now.